core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sink/CacheSinkOpExecConfig.scala

```diff
@@ -0,0 +1,37 @@
+package edu.uci.ics.texera.workflow.operators.sink
+
+import edu.uci.ics.amber.engine.architecture.breakpoint.globalbreakpoint.GlobalBreakpoint
+import edu.uci.ics.amber.engine.architecture.deploysemantics.deploymentfilter.ForceLocal
+import edu.uci.ics.amber.engine.architecture.deploysemantics.deploystrategy.RandomDeployment
```

Comment:
instead of creating a new config, can we just use existing configs?

Reply:
The configs of cache source and cache sink need uuid: String and opResultStorage: OpResultStorage. Existing configs do not have these 2 attributes.